src/components/UserSelector.tsx

```diff
@@ -20,21 +51,18 @@
 
       <div className="dropdown-menu" id="dropdown-menu" role="menu">
         <div className="dropdown-content">
-          <a href="#user-1" className="dropdown-item">
-            Leanne Graham
-          </a>
-          <a href="#user-2" className="dropdown-item is-active">
-            Ervin Howell
-          </a>
-          <a href="#user-3" className="dropdown-item">
-            Clementine Bauch
-          </a>
-          <a href="#user-4" className="dropdown-item">
-            Patricia Lebsack
-          </a>
-          <a href="#user-5" className="dropdown-item">
-            Chelsey Dietrich
-          </a>
+          {users.map(user => (
+            <a
+              href={'#user-' + user.id}
+              key={user.id}
+              className={classNames('dropdown-item', {
+                'is-active': selectedUser?.id === user.id,
+              })}
+              onMouseDown={() => handleUserSelect(user)}
```

Comment:
You are using arrow function in the `onMouseDown` handler. It's not a mistake but it's a bad practice because it may cause unnecessary re-renders. It's better to define a method for this.